templates/download/risc-v/sifive-unmatched.html

```diff
@@ -34,7 +34,7 @@ <h3 class="p-heading--5 u-no-padding--top">
         <a class="p-button--positive"
            href="https://cdimage.ubuntu.com/releases/24.10/release/ubuntu-24.10-preinstalled-server-riscv64+unmatched.img.xz">Download 24.10</a>
         <a class="p-button--positive"
-           href="https://cdimage.ubuntu.com/releases/24.04/release/ubuntu-24.04-preinstalled-server-riscv64+unmatched.img.xz">Download 24.04.1 LTS</a>
+           href="https://cdimage.ubuntu.com/releases/24.04.1/release/ubuntu-24.04.1-live-server-riscv64.img.gz">Download 24.04.1 LTS</a>
```

Comment:
Is this expected that "preinstalled server" is replaced by "live server"?

Reply:
No, I updated it in the wrong place. Thanks for catching!